### PR TITLE
houdoku: add version 2.9.2

### PR DESCRIPTION
**Pull request by SonaliBendre**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/houdoku.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.9.2",
+    "description": "Manga reader and library manager for the desktop",
+    "homepage": "https://houdoku.org/",
+    "license": "MIT",
+    "url": "https://github.com/xgi/houdoku/releases/download/v2.9.2/Houdoku-Setup-2.9.2.exe#/dl.7z",
+    "hash": "sha512:db535de9ac5de28a5d4e7639591be440e166542aa07066829dc7c81f3e0bbb4e3da54a31b031b3a262935ceb67d355da6d32d44e6eb926cb2d5cd10eb64e733c",
+    "pre_install": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+    "post_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse -Force",
```
Comment (issaclin32) [inline bucket/houdoku.json:9]:
```suggestion
    "pre_install": [
        "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
        "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse -Force"
    ],
```

**Comment by issaclin32** [comment]
config location: `$Env:AppData\Houdoku`
(This is a reminder. No need to modify anything regard to this.)

**Review by issaclin32 — APPROVED** [review]

File: bucket/houdoku.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.9.2",
+    "description": "Manga reader and library manager for the desktop",
+    "homepage": "https://houdoku.org/",
+    "license": "MIT",
+    "url": "https://github.com/xgi/houdoku/releases/download/v2.9.2/Houdoku-Setup-2.9.2.exe#/dl.7z",
+    "hash": "sha512:db535de9ac5de28a5d4e7639591be440e166542aa07066829dc7c81f3e0bbb4e3da54a31b031b3a262935ceb67d355da6d32d44e6eb926cb2d5cd10eb64e733c",
```
Comment (sitiom) [inline bucket/houdoku.json:7]: Houdoku is 64-bit so it should be wrapped inside an `architecture` block. Fixed in https://github.com/ScoopInstaller/Extras/pull/9868

**Pull request merged**